src/components/modal/modal.js

```diff
@@ -396,6 +401,11 @@ export const BModal = /*#__PURE__*/ Vue.extend({
         hide: this.hide,
         visible: this.isVisible
       }
+    },
+    ignoreEnforceFocusSelectors() {
+      const selector = this.ignoreEnforceFocusSelector
+      const selectors = isArray(selector) ? selector : [selector]
```

Comment:
A simple way to do this is with the `concat` array utility:

```js
return concat(this.ignoreEnforceFocusSelector).filter(identity).join(', ')
```

Reply:
Didn't went for the `.join(', ')` here because we could get an empty selector which `closest()` would complaint about.